src/invidious/routes/images.cr

```diff
@@ -61,27 +41,14 @@ module Invidious::Routes::Images
       end
     end
 
-    request_proc = ->(response : HTTP::Client::Response) {
-      env.response.status_code = response.status_code
-      response.headers.each do |key, value|
-        if !RESPONSE_HEADERS_BLACKLIST.includes?(key.downcase)
-          env.response.headers[key] = value
-        end
-      end
-
+    # A callable proc to be used inside #proxy_image
+    callable_proc = ->(env : HTTP::Server::Context) {
```

Comment:
Is there a reason for keeping Procs and not using Blocks?

Reply:
None, actually.